Bio/GenBank/__init__.py

```diff
@@ -1099,6 +1099,12 @@ def location(self, content):
 
         cur_feature = self._cur_feature
 
+        # Check if the sequence is circular for features that span the origin
+        seq_is_circular = (
+            "topology" in self.data.annotations
+            and self.data.annotations["topology"] == "circular"
+        )
+
```

Comment:
That can probably be shortened to:

```python
seq_is_circular = self.data.annotations.get("topology","") == "circular"
```

And perhaps ``is_circular`` is enough for the variable name?

Reply:
However, the changes appears to have lost the ``.lower()`` and use of ``in`` we had before (e.g. things like "Circular DNA" would have been accepted).

So maybe this?:

```python
seq_is_circular = "circular" in self.data.annotations.get("topology","").lower()
```

Reply:
Hi @peterjc, thanks for the quick revision. I have added those changes in the last commit.